source/reference/kubectl-minio-plugin/kubectl-minio-init.rst

```diff
@@ -18,7 +18,8 @@ Description
 
 .. start-kubectl-minio-init-desc
 
-The :mc:`kubectl minio init` command initializes the MinIO Operator. 
+The :mc:`kubectl minio init` command initializes the MinIO Operator.
+You can also use this command to upgrade an existing Operator installation to the version of the plugin.
```

Comment:
This sentence reads awkward to me.
Maybe strike the last part `to the version of the plugin.`?

> You can also use this command to upgrade an Operator installation to a later version that matches the plugin.

I dunno. But that's a suggestion.